tests/test_cirq_backend.py

```diff
@@ -370,9 +369,9 @@ def test_invalid_n_shots_in_sim_backends(cirq_backend: _CirqSimBackend) -> None:
         CirqCliffordSimBackend(),
     ],
 )
-def test_backend_info_is_set_and_characterisation_is_none(
+def test_backend_info_and_characterisation_are_none(
     cirq_backend: _CirqBaseBackend,
 ) -> None:
     b = cirq_backend
-    assert isinstance(b.backend_info, BackendInfo)
-    assert b.characterisation is None
+    assert b.backend_info == None
+    assert b.characterisation == None
```

Comment:
```suggestion
    assert b.backend_info is None
    assert b.characterisation is None
```
We should keep this change, since it's a more reliable test than `==`.